Application/Ed-Fi-Ods.sln

```diff
@@ -101,6 +101,14 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.Ods.WebApi.CompositeSp
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.Ods.WebApi", "EdFi.Ods.WebApi\EdFi.Ods.WebApi.csproj", "{538782E7-25DC-494D-902D-3EE2EC839DC6}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.Ods.Extensions.Homograph", "..\..\Ed-Fi-Extensions\Extensions\EdFi.Ods.Extensions.Homograph\EdFi.Ods.Extensions.Homograph.csproj", "{7F95B053-3122-4B81-BA53-0877DD1D8473}"

```

Comment:
This is adding a dependency on another repo, I believe the intent is for the Profiles sample to move over to Ed-Fi-Extensions, not for us to create dependencies on it.  The idea would be for the profile project to build over there independently similar to how extensions work, but I believe that work is slated for ODS-4644.  I would expect changes for this ticket (ODS-4628) to just be around fixes to support a profiles plugin, not adding samples back in or checking in any work you had to do to create a manual plugin.

Reply:
Removed .  Cover it in ODS-464